src/test/regress/sql/failure_create_distributed_table_concurrently.sql

```diff
@@ -15,6 +15,11 @@ SET citus.shard_replication_factor TO 1;
 SET citus.max_adaptive_executor_pool_size TO 1;
 SELECT pg_backend_pid() as pid \gset
 
+SELECT nextval('pg_catalog.pg_dist_shardid_seq') AS last_shard_id \gset
```

Comment:
we normally do not follow such pattern in the tests? Why is that necessary?

Reply:
Test would be flagged as flaky when we do not restore sequence number.